### ISSUE TITLE
Api and mobile access - client data

### ISSUE THREAD

**Issue by kaktux** [FreshRSS/FreshRSS#2041]
Hi there,

i just installed freshrss on a webhost and imported all my feeds. It went smoothly so far.

Now i am trying to get access with external software - according to:
https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html

For starters i installed feeedreader via flatpak on a kubuntu 18.04.

But so far i couldn't get it to access the freshrss installation.
What i did:
Under Settings->Profile
i opened the link:
https://freshrss.mydomain.com/p/api/

- From there i took the url:
Your API address:
    https://freshrss.mydomain.com/p/api/fever.php
- For username i used the one i use for freshrss login
- For passwort i tried to create one under Settings->Profile-> Password-api

But - when i do the last step= entering a password - i get an error:
it redirects to
https://freshrss.mydomain.com/p/i/?c=error
saying

Error 403 - Forbidden
You don't have the credentials to access this page (translated - so might not be 1to1).

The error is the same for both users i created (one beeing admin, the other my normal user).

So basically i am not sure why this error occurs - but this beeing the first question - the second would be if these 3 things (api adress. username, api-password) would be the right ones needed for mobile access/ tryout with feedreader.

**Comment by Alkarex** [comment]
Hello,
[FeedReader](https://jangernert.github.io/FeedReader/) is using the Google Reader API, not the Fever API.
Try instead with `https://freshrss.mydomain.com/p/api/greader.php`

Regarding setting the API password, there has been problems with some password managers, which wrongly put some values in the first password field, so watch out for that.

**Comment by math-GH** [comment]
What is left todo here?

**Comment by Frenzie** [comment]
Fixed by https://github.com/FreshRSS/FreshRSS/pull/2942.
